packages/browser-history/package.json

```diff
@@ -15,18 +15,11 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./dist/browser-history.cjs.js",
-  "module": "./dist/browser-history.es.js",
-  "typings": "./dist/typings/index.d.ts",
-  "types": "./dist/typings/index.d.ts",
```

Comment:
Don't we need these for consumers of the package? At least types? 🤔

Reply:
Apparently TypeScript can still find the declarations, according to this: https://github.com/preconstruct/preconstruct/pull/93#pullrequestreview-285817702